R/fit.R

```diff
@@ -273,10 +274,15 @@ mmrm <- function(formula,
                  reml = TRUE,
                  optimizer = "automatic",
                  n_cores = 1L,
-                 accept_singular = TRUE) {
+                 accept_singular = TRUE,
+                 method = c("Satterthwaite", "Kenward-Roger", "Kenward-Roger-Linear", "ST", "KR", "KRLin")) {
```

Comment:
```suggestion
                 method = c("Satterthwaite", "Kenward-Roger", "Kenward-Roger-Linear")) {
```
let's go for the long method names - or for the short names. see below